optimade/server/models/toplevel.py

```diff
@@ -55,3 +56,9 @@ class OptimadeStructureResponse1(OptimadeResponse1):
 
 class OptimadeStructureResponseMany(OptimadeResponseMany):
     data: List[StructureResource]
+
+
+class OptimadeErrorResponse(BaseModel):
```

Comment:
The spec says: 

> If there were errors in producing the response all other fields MAY be skipped, and the following field MUST be present: errors

So, I guess, `meta` would also be optional.
And there's also `included` that's missing from above, right?
Do we actually need a subclass for this?
Can't we simply add `errors` as an optional field in `OptimadeResponse1` and `OptimadeResponseMany`?

Reply:
Right, meta would be optional as well. I'm not sure we need a subclass either. I would like to merge just so we get `optimade/server/models/errors.py` in and then follow-up PRs can refactor / correct misreadings of the spec.